server/notification-providers/notifery.js

```diff
@@ -0,0 +1,59 @@
+const { getMonitorRelativeURL, UP } = require("../../src/util");
+const { setting } = require("../util-server");
+const NotificationProvider = require("./notification-provider");
+const axios = require("axios");
+
+class Notifery extends NotificationProvider {
+    name = "notifery";
+
+    /**
+     * @inheritdoc
+     */
+    async send(notification, msg, monitorJSON = null, heartbeatJSON = null) {
+        const okMsg = "Sent Successfully.";
+        const url = "https://api.notifery.com/event";
+
+        // Prepare request data
+        let data = {
+            title: notification.notiferyTitle || "Uptime Kuma Alert",
+            message: msg,
+        };
+
+        // Add group if specified
+        if (notification.notiferyGroup) {
+            data.group = notification.notiferyGroup;
+        }
+
+        // Add link to monitor
+        const baseURL = await setting("primaryBaseURL");
+        if (baseURL && monitorJSON) {
+            // Add monitor link as part of the message
+            data.message += `\n\nMonitor: ${baseURL}${getMonitorRelativeURL(monitorJSON.id)}`;
+        }
+
+        // Add code and duration if heartbeat exists
+        if (heartbeatJSON) {
+            // Add status code - if DOWN use code 1, otherwise 0
+            data.code = heartbeatJSON.status === UP ? 0 : 1;
+
+            // Add ping duration if available
+            if (heartbeatJSON.ping) {
+                data.duration = heartbeatJSON.ping;
+            }
+        }
```

Comment:
I like to only add comments whree they add context, otherwise this feels too much like grafiti

```suggestion
        let data = {
            title: notification.notiferyTitle || "Uptime Kuma Alert",
            message: msg,
        };

        if (notification.notiferyGroup) {
            data.group = notification.notiferyGroup;
        }

        // Link to the monitor
        const baseURL = await setting("primaryBaseURL");
        if (baseURL && monitorJSON) {
            data.message += `\n\nMonitor: ${baseURL}${getMonitorRelativeURL(monitorJSON.id)}`;
        }

        if (heartbeatJSON) {
            data.code = heartbeatJSON.status === UP ? 0 : 1;

            if (heartbeatJSON.ping) {
                data.duration = heartbeatJSON.ping;
            }
        }
```